app/views/shared/_show_breadcrumbs.html.erb

```diff
@@ -1,20 +1,26 @@
 <% parents = Arclight::Parents.from_solr_document(document).as_parents %>
+<% h1_index = 0 %>
```

Comment:
I'm not sure I understand the significance of the `h1_index`  and how's it's being used. I guess it's also unclear to me why we need to build it up inside the loop below instead of calculating it immediately?

Reply:
We are using index to track the level of indentation need in the breadcrumb display. This is based stepwise on the last indentation. Once we are out of the loop the index value is lost so h1_index has a larger scope. I could call last_index and then do the addition just before the h1 element.

Does that answer your question? Am I missing something?

Reply:
I guess; then isn't `h1_index` just `parents.length + 3`?

Reply:
Hey, maybe it is. I'll try that!

Reply:
> <%= content_tag :h1, class: "breadcrumb-item breadcrumb-item-#{parents.length + 2} media" do %>

Nice!